### PR TITLE
readme npm/yarn updates with tests

### PR DESCRIPTION
**Pull request by kellyselden**
cc @Turbo87 mentioned here https://github.com/ember-cli/ember-cli/pull/7177#discussion_r125192909

### CODE REVIEW

File: blueprints/app/files/README.md [hunk 1]
```diff
@@ -8,15 +8,16 @@ A short introduction of this app could easily go here.
...
+* [Yarn](https://yarnpkg.com/)<% } else { %> (with NPM)<% } %>
 * [Ember CLI](https://ember-cli.com/)
 * [Google Chrome](https://google.com/chrome/)
 
 ## Installation
 
 * `git clone <repository-url>` this repository
 * `cd <%= name %>`
-* `npm install`
+* `<% if (yarn) { %>yarn<% } else { %>npm<% } %> install`
```
Comment (Turbo87) [inline blueprints/app/files/README.md]: I prefer yarn install to be more explicit

Comment (kellyselden) [inline blueprints/app/files/README.md]: How so?

Comment (Turbo87) [inline blueprints/app/files/README.md]: nevermind, apparently that's already what you're doing... review on mobile is not a good idea :D

**Comment by kellyselden** [comment]
Any opposition?

**Pull request merged**